packages/components/src/SectionMessage/SectionMessage.tsx

```diff
@@ -83,6 +92,8 @@ export const SectionMessage = ({
   size,
   children,
   closeButton,
+  dismissable,
```

Comment:
I think I'm not really happy with `dismissable` and `onDismiss`. I compared it to Dialog and there we use `dismissable` to avoid closing the dialog when clicking outside. Here it is another usage for this prop. To keep our props or API consistent and maybe also the user wouldn't expect this I would prefer to rename them. So my proposal is `dismiss` = `open` and `onDismiss` = `onOpenChange`. What do you think? @marigold-ui/developer

Reply:
I think you can not open and close a SectionMessage, this was also why i was not sure how it is meant in the SUP

Reply:
With `open` I agree. But than maybe `close`. At least the user can click a close button and it affects the state.

That's whats ChatGPT says:

<strong>Key Differences</strong></h3>
Feature | dismiss | open / close
-- | -- | --
Type | Action (callback function) | State or event prop (boolean/handler)
Responsibility | Notifies the parent to dismiss | Controls visibility directly
Parent's Role | Handles dismissal logic | Manages component state
Child's Role | Calls the callback to dismiss | Reacts to state changes
Flexibility | More decoupled and reusable | Clearer control of visibility

Reply:
Hm I at first had it like this, if its ok for @sebald  i can change it back to `close` and `onClose` https://reservix.atlassian.net/browse/DSTSUP-98?focusedCommentId=453484

Reply:
yeah, that sounds fine with me. Maybe a personal preferrence, but you open/close a dialog, but you dismiss a message :)